src/app/results/xyz.html

```diff
@@ -0,0 +1,75 @@
+<app-navbar></app-navbar>
+<div class="row">
+    <div class="container-fluid" id="search-options-field">
+        <ul type="none" id="search-options">
+            <li [class.active_view]="Display('all')" (click)="docClick()">All</li>
```

Comment:
@13nehu why did you changed the file name to xyz.html ? Please, let the file name remain same. It makes it easy for every to find the code of the component.